src/averageAges.js

```diff
@@ -15,11 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person['sex'] === 'm');
```

Comment:
Don't repeat yourself: just check at the start of the function if century argument exists
if yes - you should create the array with men for correct century
if no - you should create the array with all men

and then logic for both of variants is the same: reducing and getting the average value
```suggestion
  const men = century
    ? people.filter(...)
    : people.filter(person => person['sex'] === 'm');
```